### PR TITLE
UIORG-25 sso config

### PR DESCRIPTION
**Pull request by szkata**
Updated metadata file download to use stripes-connect.

### CODE REVIEW

**Comment by szkata** [comment]
@mkuklis I made the changes you suggested. Could you take a look and merge if it looks good?

File: settings/SSOSettings.js [hunk 1]
```diff
@@ -59,17 +57,20 @@ class SSOSettings extends React.Component {
   }
 
   updateSettings(settings) {
-    this.props.mutator.samlconfig.PUT(settings).then((result) => {
-      this.props.stripes.logger.log('Response: ', result);
+    this.props.mutator.samlconfig.PUT(settings);
+  }
+
+  downloadMetadata(callback) {
+    this.props.mutator.downloadFile.GET().then((result) => {
```
Comment (mkuklis) [inline settings/SSOSettings.js]: Could you please add:
`this.props.mutator.downloadFile.reset();` before executing `this.props.mutator.downloadFile.GET()`

**Comment by mkuklis** [comment]
@szkata this looks great! I see a conflicts now though. Would you mind resolving those and then I will merge. Thank you!

**Comment by mkuklis** [comment]
@szkata never mind I think I got those resolved. I will just merge now. Cheers!

**Pull request merged**